src/applications/mhv/secure-messaging/tests/components/Compose/ComposeForm.unit.spec.jsx

```diff
@@ -168,6 +146,18 @@ describe('Compose form component', () => {
     expect(deleteButton).to.exist;
   });
 
+  it('renders without errors on send button click', async () => {
+    const screen = setup(draftState, `/draft/${draftMessage.id}`);
+
+    fireEvent.click(screen.getByTestId('Send-Button'));
```

Comment:
Is this test missing an assertion?  Same for the next few tests right after this one too.

Reply:
yeah, probably. let me look into that. good catch